app/src/components/authentication/SignUpForm.tsx

```diff
@@ -97,45 +93,22 @@ export const SignUpForm = ({
       >
         Sign up
       </Button>
-      <Collapse sx={{width: '100%'}} in={errorMessage !== ''}>
-        <Alert
-          severity="error"
-          action={
-            <IconButton
-              aria-label="close"
-              color="inherit"
-              size="small"
-              onClick={() => {
-                setErrorMessage('');
-              }}
-            >
-              <CloseIcon fontSize="inherit" />
-            </IconButton>
-          }
-        >
-          {errorMessage}
-        </Alert>
-      </Collapse>
       <Divider>or</Divider>
       <Button
+        disabled={getTokenIsLoading}
         variant="outlined"
-        color="secondary"
         size="large"
         fullWidth
+        sx={{color: 'text.primary'}}
         // overrides the default react router link since we're hitting a redirect from the api
         component="a"
         href={`/api/auth/google?redirect_uri=http://localhost:4040/signup/${type}`}
```

Comment:
Shouldn't we redirect users back to the production site, not necessarily `localhost`? Unless I'm missing something, it seems to me that the base URL should be the `.env` file's `ROOT_URL` value.

Reply:
For some reason the google authentication doesn't work unless I move the `${type}` suffix from the redirect url.